tests/test_contexts.py

```diff
@@ -35,29 +33,29 @@ def block_checks_glb(name, glbs, body, obs=None):
     block = glbs[name]
     obs = obs or {}
     for k, v in obs.items():
-        yield assert_equal, v, glbs[k]
+        assert v == glbs[k]
     if isinstance(body, str):
         body = body.splitlines()
-    yield assert_equal, body, block.lines
-    yield assert_is, glbs, block.glbs
-    yield assert_is, None, block.locs
+    assert body == block.lines
+    assert glbs is block.glbs
+    assert block.locs is None
```

Comment:
nope it will stop at the first `assert False` in each individual test, pytest favors the style of writing  tests with few or even one assert.